webofneeds/won-node/src/main/resources/db/migration/V1__initial_database.sql

```diff
@@ -0,0 +1,118 @@
+BEGIN;
+CREATE TABLE bastate
```

Comment:
Maybe we can put here only these tables and objects that are really used by the wonnode and leave away the tables and objects of the owner in this file